src/org.openjdk.nashorn/share/classes/org/openjdk/nashorn/internal/runtime/Context.java

```diff
@@ -829,7 +829,7 @@ public Object eval(final ScriptObject initialScope, final String string,
 
         Class<?> clazz;
         try {
-            clazz = compile(source, new ThrowErrorManager(), strictFlag, true);
+            clazz = compile(source, new ThrowErrorManager(), strictFlag);
```

Comment:
I vaguely isEval or not was used in compilation to compile differently. Or perhaps only for loading post compilation? (eval code as throw-away anon class?). But if the current code does not use it, then fine.